pkgs/applications/video/obs-studio/obs-move-transition.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, cmake
+, obs-studio
+}:
+
+let
+  findlibobs = fetchurl {
+    url = "https://raw.githubusercontent.com/obsproject/obs-studio/master/cmake/external/FindLibobs.cmake";
```

Comment:
this url is not stable and the hash will soon be invalidated.

You can probably fetch this file as `${obs-studo.src}/cmake/external/FindLibobs.cmake`, or even better, install this file in `postInstall` in obs-studio and then cmake might find it by itself?

Reply:
I went with the first suggestion because I am not familiar with the second one.